app/views/summary/index.html.haml

```diff
@@ -0,0 +1,43 @@
+%div.inset
+  %table.table.table-striped.table-bordered.table-responsive
+    %caption
+      %span.namespace= @summary.client_namespace
+      Requests: Fiscal Year
+      = @summary.fiscal_year
```

Comment:
Since this endpoint is not (yet) linked to from anywhere, on purpose, I expected we would get some user feedback before we made it more "formal".

I added the `:fiscal_year` option in order to make it easier to rule-of-thumb benchmark the effect of profiling an entire year's worth of proposals in one request, since there is a SQL execution for each one.